tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/common/types/access/jdt/WorkingCopyOwnerProviderTest.java

```diff
@@ -102,7 +102,9 @@ public void testWorkingCopyOwber_02() {
       StringConcatenation _builder_1 = new StringConcatenation();
       _builder_1.append("package foo;");
       _builder_1.newLine();
-      _builder_1.append("public class MyClass{}");
+      _builder_1.append("public static class MyClass{");
```

Comment:
This is invalid java. The stub generator should only add the static modifier for the nested types.